docs/admin/deployment.rst

```diff
@@ -117,13 +117,27 @@ you create:
    Username for the SMTP connection (default: ``'apikey'``).
 
 ``EMAIL_PORT``
-   SMTP port (default: ``587``)
+   SMTP port (default: ``587``).
 
 ``EMAIL_USE_TLS``
-   Use explicit TLS for the SMTP connection (default: ``True``)
+   Use explicit TLS for the SMTP connection (default: ``True``).
 
 ``EMAIL_USE_SSL``
-   Use implicit TLS for the SMTP connection (default: ``False``)
+   Use implicit TLS for the SMTP connection (default: ``False``).
+
+``EMAIL_CONSENT_ENABLED``
+   Enables Email consent page (default: ``False``).
+
+``EMAIL_CONSENT_TITLE``
+   Optional. Title of the Email consent page.
+
+``EMAIL_CONSENT_MAIN_TEXT``
+   Optional. Main text of the Email consent page. You can use that to explain what type
+   of communication to expect among other things.
+
+``EMAIL_CONSENT_PRIVACY_NOTICE``
+   Optional. Privacy notice on the Email consent page. It's possible to use HTML and
+   link to external privacy notice page.
```

Comment:
I'm not sure that "Optional" is really right, as I presume that these are required if `EMAIL_CONSENT_ENABLED` is set?

Reply:
Technically it's optional, because `EMAIL_CONSENT_ENABLED` is also optional. I'll make that clearer.